docs/developer-docs/latest/setup-deployment-guides/configurations/optional/environment.md

```diff
@@ -22,6 +22,7 @@ Strapi provides the following environment variables:
 | `ENV_PATH`                                                 | Path to the file that contains your environment variables                                                                                                                                                                                                                                                  | `String`  | `'./.env'`      |
 | `STRAPI_PLUGIN_I18N_INIT_LOCALE_CODE` <br/><br/>_Optional_ | Initialization locale for the app, if the [Internationalization (i18n) plugin](/developer-docs/latest/plugins/i18n.md) is installed and enabled on Content-Types (see [Configuration of i18n in production environments](/developer-docs/latest/plugins/i18n.md#configuration-of-the-default-locale)) | `String`  | `'en'`          |
 | `API_TOKEN_SALT`<br/><br/>_Optional_                       | Salt to use to generate [API tokens](/developer-docs/latest/setup-deployment-guides/configurations/optional/api-tokens.md)                                                                                                                                                                                 | `String`  | -               |
+| `APP_KEYS`                       | static keys used by the [session middleware](/developer-docs/latest/setup-deployment-guides/configurations/required/middlewares.md#internal-middlewares-configuration-reference)     | `String`    |   `undefined`   
```

Comment:
Actually we should not put it as a config variable. And once https://github.com/strapi/strapi/pull/12287 is merged, we should also remove the line 24 (`API_TOKEN_SALT`).
Are you available to discuss about it, it would be easier on face-to-face :)